ci-operator/step-registry/cucushift/installer/check/vsphere/cpms/cucushift-installer-check-vsphere-cpms-commands.sh

```diff
@@ -8,7 +8,7 @@ export KUBECONFIG=${SHARED_DIR}/kubeconfig
 
 
 
-function checkMultizone() {
+function checkcpms() {
```

Comment:
pls also change the following `INFO` message in this function to remove the `multi` keyword.

Reply:
got it.